swift_browser_ui_frontend/src/components/CFooter.vue

```diff
@@ -82,7 +82,9 @@ export default {
 <style lang="scss" scoped>
 @import "@/css/prod.scss";
 
-#c-footer{
+footer{
+  position:relative;
```

Comment:
this should not be required if the whole page is under display flex :thinking: need to take a look

Reply:
Good point, I'll take a look

Reply:
Indeed, it doesn't really need that.